src/main/java/com/synopsys/integration/alert/web/security/authentication/saml/AlertSAMLEntryPoint.java

```diff
@@ -54,6 +51,8 @@ public void commence(final HttpServletRequest request, final HttpServletResponse
             super.commence(request, response, e);
             return;
         }
-        throw new ServletException(e);
+        if (e instanceof InsufficientAuthenticationException) {
```

Comment:
Do we want to do anything with the exception if this returns false?

Reply:
I do not believe so the ServletException was a way to force our UI to display the login screen but the log would be spammed with the exception string.  This is handled in a filter chain so we don't want the exception to bubble up.  It was bubbled up to force the Login Screen to show up because SAML wasn't enabled.  This is a better solution and I would want to make this single file change for 4.1.0 after this is review.